source/installing-tarball.rst

```diff
@@ -0,0 +1,65 @@
+:orphan:
+
+.. _proxysql-admin-interface.installing-tarball:
+
+Installing from a tarball
+================================================================================
+
+Installing |proxysql| from a tarball is an alterative method if recommended
+approaches, using apt or yum package managers, are not applicable in your
+environment. With this method, you download a |tar-file| file from the `ProxySQL
+v1 downloads page`_ or from `ProxySQL v2 downloads page`_ and then install to a
```

Comment:
For tarballs, we can point tarball package link directly using https://www.percona.com/downloads/proxysql/proxysql-1.4.16/binary/
https://www.percona.com/downloads/proxysql2/proxysql2-2.0.12/binary/